app/model/editions/client/EditionsClientCollection.scala

```diff
@@ -109,14 +109,14 @@ object EditionsSupportingClientCard {
 
   def toFeastCollectionItem(supportingCard: EditionsSupportingClientCard) =
     supportingCard.cardType match {
-      case Some(CardType.Recipe) =>
-        EditionsRecipe(supportingCard.id, supportingCard.frontPublicationDate)
       case Some(CardType.Chef) =>
         EditionsChef(
           supportingCard.id,
           supportingCard.frontPublicationDate,
           supportingCard.meta.map(_.toChefMetadata)
         )
+      case _ =>
```

Comment:
Hi @davidfurey, thanks for asking, 
If I remember, we do need case matching for Recipe and Chef only, we don't want this to work for Feast-Collection(FC), because we don't have yet supporting feature of FC (present inside another FC) to drag into some another FC.
Can we have undefined or something similar if we really need to do case _?

Reply:
Thanks for discussion @davidfurey to discuss on ways to solve this. 
I will make change in later PR and attach here as reference.